scripts/Units/ParticipantsUnit.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using UnityEngine;
+using Unity.VisualScripting;
+using DolbyIO.Comms;
+using DolbyIO.Comms.Unity;
+
+namespace DolbyIO.Comms.Unity
+{
+    [UnitTitle("Get Participants")]
+    [UnitCategory("DolbyIO")]
+    public class ParticipantsUnit : Unit
+    {
+        private DolbyIOSDK _sdk = DolbyIOManager.Sdk;
+
+        [DoNotSerialize]
+        public ValueInput ParticipantIds;
+
+        [DoNotSerialize]
+        public ValueOutput Participants;
+
+        protected override void Definition()
+        {
+            ParticipantIds = ValueInput(nameof(ParticipantIds), new string[0]);
+            Participants = ValueOutput<List<Participant>>(nameof(Participants), GetParticipants);
+        }
+
+        private List<Participant> GetParticipants(Flow flow)
+        {
+            var participants = _sdk.Conference.GetParticipantsAsync().Result;
+            string[] ids = flow.GetValue<string[]>(ParticipantIds);
+
+            if (ids.Length > 0)
```

Comment:
I like `.Any()` here, what do you think?

Reply:
Any is good, but I don't want to include Linq, and it adds overhead. Even very small, I prefer this method when doing stuff for game engines :)